cortex/utils.py

```diff
@@ -1030,13 +1030,16 @@ def download_subject(subject_id='fsaverage', url=None, pycortex_store=None,
     print('Downloaded subject {} to {}'.format(subject_id, tmp_dir))
     # Un-tar to pycortex store
     if pycortex_store is None:
-        # Default location is config file pycortex store.
-        pycortex_store = config.get('basic', 'filestore')
+        # Default location is current filestore in cortex.db
+        pycortex_store = db.filestore
     pycortex_store = os.path.expanduser(pycortex_store)
     with tarfile.open(os.path.join(tmp_dir, subject_id + '.tar.gz'), "r:gz") as tar:
         print("Extracting subject {} to {}".format(subject_id, pycortex_store))
         tar.extractall(path=pycortex_store)
 
+    # force filestore reload
+    db._subjects = None  
```

Comment:
I'm wondering whether it will make any difference to also force repopulating the dictionary by adding a call to `_ = cortex.db.subjects` after this line. (Right now the dictionary will be None until the next call to `cortex.db.subjects`...what if some function tries to access `cortex.db._subjects` instead of `cortex.db.subjects`?)

It's probably safe to keep the code like this, but who knows if it will have unintended consequences...

Reply:
Well, `cortex.db._subjects` is not supposed to be accessed directly. Actually, it might be better to have a method `cortex.db.reload_subjects`, which reset `_subjects` internally. Will do that.